.github/workflows/pypi-publish.yml

```diff
@@ -16,8 +16,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install setuptools wheel twine
-        python setup.py sdist bdist_wheel
+        python -m pip install setuptools wheel twine
+        python -m pip wheel
```

Comment:
What's all of this for? It looks like you're dropping setuptools, so why install it? And why do you need to explicitly install wheel?

Reply:
Guessing just old habits.

Once this is `pyproject.toml` based, the `pip wheel` command will install the dependencies listed in that file and they don't need to be installed first. So `twine` is the only one you'll need to install manually.

Reply:
Old habbits is a nice way to put it. But these points are exactly why i asked you both to review 😀

Reply:
```suggestion
        python -m pip install wheel twine
```

Reply:
i think i will try and switch this to flit?